### ISSUE TITLE
EclipseFP is typing slow. (What I've found)

### ISSUE THREAD

**Issue by Zenhoth** [JPMoresmau/eclipsefp#289]
EclipseFP is nice, but it's FAR TOO convenient. It's annoying that everytime I save my files the console (ghc) automatically compiles the code ready for use, but this is not what I'd like because I want to keep saving/compiling separate. Another thing is that rubbish tool called 'BuildWrapper' which provides syntax highlighting and that slow typing. I think what's happening is that everytime I type, the editor is attempting to compile whatever is being written in real-time. This is not convenient because when you're typing the code may not even be terminated, and then GHC ends up reporting errors. 

The solution (for now) to the slow typing is to disable BuildWrapper through Window > Preferences > Haskell and then typing a random executable (or mistyping the foresaid) so that it is not used at all. 

In the meantime, I propose dismantling BuildWrapper so that the following is addressed:

1. Stop compiling code in real-time
3. Stop having the editor constantly synchronising especially when the file isn't saved

TL;DR: disable BuildWrapper to disable the slow typing (which after looking through threads on here seems to have not been resolved, which is obviously a bug :-1: )) You can still compile your code as normal.

**Comment by flomerz** [comment]
I had exactly the same problem with the newest ghc version. Older versions work. I would like to see an update of the EclipseFP, because i think it's a very good IDE for programming haskell. I don't like console editor like emacs or vi for programming. I also tried a Leksah, Sublime Plugin or Atom Plugin, but they are not competitive to Eclipse.

**Comment by JPMoresmau** [comment]
Well I've stopped maintaining it, so take over. And people calling the tooling "rubbish" but not contributing anything are actually one of the reason I stopped maintaining it. Tired of getting abused by people too lazy to contribute...
